database/migrations/2023_09_21_171540_adding_column_to_assigments_student.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up()
+    {
+        Schema::table('assignment_students', function (Blueprint $table) {
+            $table->string('assigment_file_path');  // Assuming you want to add a string column
+        });
+        Schema::table('assignment_students', function (Blueprint $table) {
+            $table->string('assigment_description');  // Assuming you want to add a string column
```

Comment:
Make the name of this column more descriptive. assignmnet_student_comment. Also currently there is a typo in the word assignment.